lib/home/view/fab_view.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+class floatingActionButtonView extends StatelessWidget {
+  const floatingActionButtonView({
+    super.key,
+  });
+
+  @override
+  Widget build(BuildContext context) {
+    return FloatingActionButton(
+      onPressed: () {},
+      child: const Icon(Icons.add),
+    );
+  }
+}
```

Comment:
Few things here:
1) Floating Action Button is a widget, not view, so please place it in the home widgets folder instead of view (screen)
2) Class name should start from capital letter 
3) you can store it in tasks_page.dart as the private class (you should determine if you're gonna use it in multiple places, if not, store it as a private class in the page.